src/quo/components/wallet/address_text/schema.cljs

```diff
@@ -10,5 +10,11 @@
       [:format {:optional true} [:enum :short :long]]
       [:theme :schema.common/theme]
       [:networks {:optional true}
-       [:maybe [:sequential [:map [:network-name :keyword] [:short-name :string]]]]]]]]
+       [:maybe [:sequential [:map [:network-name :keyword] [:short-name :string]]]]]
+      [:full-address? {:optional true} [:maybe :boolean]]
+      [:size {:optional true} [:maybe :keyword]] ;; TODO: it's text schema and should be implemented in
```

Comment:
can we create an issue for the TODO and link it in that comment?

Reply:
Sure

Reply:
Done